src/lem/tag.rs

```diff
@@ -32,7 +33,9 @@ impl Tag {
 impl Display for Tag {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         match self {
+            Tag::Nil => write!(f, "Nil"),
             Tag::Dummy => write!(f, "Dummy"),
```

Comment:
Just commenting since I see this now: we should *not* abuse tags this way (cc: @johnchandlerburnham).

Reply:
This `symbol.rs` file will disappear. This shim is temporary, while the mega PR isn't merged

Reply:
How come?